transactions/FlowServiceAccount/set_tx_fee_surge_factor.cdc

```diff
@@ -9,6 +9,6 @@ transaction(surgeFactor: UFix64) {
 			?? panic("Unable to borrow reference to administrator resource")
 	}
 	execute {
-		self.flowFeesAccountAdmin.setFeeSurgeFactor(surgeFactor: surgeFactor)
+		self.flowFeesAccountAdmin.setFeeSurgeFactor(surgeFactor)
```

Comment:
label not needed: 
https://github.com/onflow/flow-core-contracts/blob/fde69782f6d07fb07649763ae0912394b9d261d0/contracts/FlowFees.cdc#L51